Client/src/Pages/Home.jsx

```diff
@@ -1,9 +1,18 @@
-import NewsAPI from "../Components/NewsAPI";
+import { AltTextProvider} from '../Components/EventCreation/AltTextContext';
+import NewsAPI from '../Components/NewsAPI';
+import EventForm from '../Components/EventCreation/EventForm';
+
 
 export default function Home() {
     return (
         <>
-        <p> Home Page </p>
-        <NewsAPI /></>
+            <AltTextProvider>
+                <div className='row-container'>
+                <EventForm/>
+                <NewsAPI/>
+                </div>
+            </AltTextProvider>
```

Comment:
Is the Alt text provider something you only want to be accessible to the components of the home page? Is Alt text something you could also use to improve accessibility for other pages such as the user profile or event page?

Reply:
I think it's just for the eventpage for now, but I might integrate it to other pages.